### PR TITLE
adds client license for lapack utils code

### PR DESCRIPTION
**Pull request by TorreZuk**
* follows same pattern as rocBLAS

### CODE REVIEW

**Review by daineAMD — APPROVED** [review]
@NaveenElumalaiAMD maybe you should take a quick look in case you made changes elsewhere too.

File: docs/install.rst [hunk 1]
```diff
@@ -148,6 +148,10 @@ Dependencies
 Dependencies are listed in the script install.sh. Use ``install.sh`` with ``-d`` option to install dependencies.
 CMake has a minimum version requirement listed in the file install.sh. See --cmake_install flag in install.sh to upgrade automatically.
 
+However, for the test and benchmark clients' host reference functions you must manually download and install AMD's ILP64 version of the AOCL libraries, version 4.1 or 4.0, from https://www.amd.com/en/developer/aocl.html.
+The aocl packages include AOCL-BLAS and AOCL-LAPACK.
```
Comment (daineAMD) [inline docs/install.rst]: AOCL should probably be capitalized here.

**Review by daineAMD — APPROVED** [review]

**Comment by TorreZuk** [comment]
Can improve later in your doc PR @daineAMD if Naveen comments, I'll merge now.

**Review by NaveenElumalaiAMD — COMMENTED** [review]
Sorry for the late reponse, checked the copyrights and it is included in both LICENSE.md and [clients/include/lapack_utilities.hpp] (https://github.com/ROCm/hipBLAS/pull/745/files#diff-a04c9bb1cf5bd70f82c981437be77e8b469bfbd56f01379166bced83e6d7f2e9)

**Pull request merged**